demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -707,6 +707,7 @@ export class DemoMeetingApp
           }
           /* @ts-ignore */
           if ((e as SubmitEvent).submitter.id == 'quick-join') {
+            await this.openAudioInputFromSelectionAndPreview();
```

Comment:
Irrelevant, but can you also make the abort on fatal error as the default option as you have this PR :)

Reply:
sure!